src/main/resources/userEmailTemplate-feedbackSession.html

```diff
@@ -61,6 +61,7 @@
     For any non-technical queries (e.g. assigned to the wrong team, misspelled name, request for deadline extension, etc.), you can contact your instructor(s) at: ${coOwnersEmails}.
 </p>
 
+${additionalInstructions}
```

Comment:
You need to update `generateFeedbackSessionEmailBasesForInstructorReminders`, which uses this template on line 79, EmailGenerator.